src/app/signup/page.tsx

```
@@ -71,69 +71,72 @@ const SignUpPage = () => {
+            helperText={
+              password.error ? '8자 이상 / 영문, 숫자, 특수문자 가능' : ''
             }
-          ]}
+            value={password.value}
+            placeholder="8자 이상 / 영문, 숫자, 특수문자 가능"
+            error={password.error ? true : false}
+            onChange={handlePwdChange}
+          />
+          <Input
+            type="password"
+            helperText={passwordCheck.error ? '비밀번호를 확인해주세요' : ''}
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)